index.html

```diff
@@ -0,0 +1,27 @@
+<html lang="en">
+      <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="stylesheet.css">
+        <title>Personal Portfolio: Omar Gutierrez</title> 
+     </head>
+     <body>
+      <header class="header">   
+        <nav class="toolbar">
+          <p class="logo">Omar Gutierrez</p>
+          <div class="hamburger"><img src="./images/Hamburger.svg" alt="menu"></div>
+        </nav>
+        <section class="headline">
+          <h1 class="greeting">Hey There. I'm Omar.</h1>
+          <h3 class="presentation">I'm a Software Developer</h3>
+          <p class="description">I will help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesistate to contact me</p>
+          <div class="socials">
+            <a href="https://www.linkedin.com/in/ommarz/" target="_blank" rel="noopener"><img src="./images/Disabled LinkedIn.svg" alt="linkedin" class="social-icon"></img></a>
+            <a href="https://github.com/ommarz" target="_blank" rel="noopener"><img src="./images/Disabled GitHub.svg" alt="github" class="social-icon"></img></a>
+            <a href="https://www.twitter.com/ommarz/" target="_blank" rel="noopener"><img src="./images/Disabled Twitter.svg" alt="twitter" class="social-icon"></img></a>
+          </div>
```

Comment:
- [ ] It is required to icons uses a `ul` tag to group your social icons, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Kindly implement this requirement.
- [ ] Also, some icons are not showing. You might need to look into this too and fix.

### Your Design
![your-social-icons](https://github.com/ommarz/Personal_Portfolio/assets/26597432/37395587-9aa8-4803-8bb5-462d07baa100)

### Figma Design
![figma-social-icons](https://github.com/ommarz/Personal_Portfolio/assets/26597432/421952a8-1a2a-409c-96df-51b537e93775)